app/controllers/lines_controller.rb

```diff
@@ -1,6 +1,11 @@
 # Single-serving controller for setting current line for a user.
 class LinesController < ApplicationController
-  def new; end
+  def new
+    if LINES.count == 1
+      session[:line] = LINES[0]
```

Comment:
cool thing you can do with arrays in ruby/rails -- `Array.first` is equivalent to `Array[0]`. 'second', 'third' might work too, I forget. Cool either way, but a nice convenience method!